ui/component/featuredBanner/view.jsx

```diff
@@ -27,6 +27,7 @@ export default function FeaturedBanner(props: Props) {
   const [pause, setPause] = React.useState(false);
   const [kill, setKill] = React.useState(false);
   const wrapper = React.useRef(null);
+  const imageWidth = width >= 1600 ? 1700 : width >= 1150 ? 1150 : width >= 900 ? 900 : width >= 600 ? 600 : 400;
```

Comment:
don't we already have code that optimizes the image properly to the screen size?

Reply:
Not really, unless I'm missing something. Wer have an `OptimizedImage` component, but you'd still have to pass the width, quality etc.. That seems kinda redundant.